scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -406,7 +422,7 @@ def unpacker(res):
             res = hypotest(*data1d, *args, nan_policy=nan_policy, **kwds)
         res_1db = unpacker(res)
 
-        assert_equal(res_1db, res_1da)
+        assert_allclose(res_1db, res_1da, 1e-15)
```

Comment:
minor: can we pass the tol as a kwarg

Reply:
Sure. If there are other things to change, I can commit that then.

Reply:
```suggestion
        assert_allclose(res_1db, res_1da, rtol=1e-15)
```